srcpkgs/qmidiarp/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'qmidiarp'
+pkgname=qmidiarp
+version=0.6.5
+revision=1
+build_style=gnu-configure
+hostmakedepends="automake libtool pkg-config qt5-host-tools"
+makedepends="alsa-lib-devel jack-devel lv2 qt5-devel"
+short_desc="MIDI arpeggiator and LFO"
+maintainer="Matthias von Faber <mvf@gmx.eu>"
+license="GPL-2.0-or-later"
+homepage="http://qmidiarp.sourceforge.net"
+distfiles="${SOURCEFORGE_SITE}/project/${pkgname}/${pkgname}/${version}/${pkgname}-${version}.tar.bz2"
+checksum=22a3a26d3dbe4bf215aa33c0fd4a79c088549328477840d00e72e50c6e807e10
+CXXFLAGS="-D_GNU_SOURCE"
```

Comment:
since you run autoeconf anyway, patch it in?

Reply:
Hm, what would be the benefit? A patch would be more verbose and messier to maintain. The envvar is straightforward and build system agnostic...